### PR TITLE
fix #100416: redesigned playback panel

### PR DESCRIPTION
**Pull request by salilkhandelwal**
The playback panel has been redesigned and is more user friendly.

### CODE REVIEW

**Comment by lasconic** [comment]
This is what I see with this PR.

capture d ecran 2016-03-16 19 11 31

**Comment by salilkhandelwal** [comment]
screenshot from 2016-03-16 20 53 31

**Comment by salilkhandelwal** [comment]
Well that is strange , I am getting as shown above , The only thing that I have differed is that I have used bold font  without italics  . I have not included the forward button as discussed on IRC .

**Comment by lasconic** [comment]
I'm on a mac, if it helps

**Comment by salilkhandelwal** [comment]
I am using ubuntu 14.04 , but that should not make a difference I guess

**Comment by salilkhandelwal** [comment]
What do you suggest then  ?

File: mscore/playpanel.ui [hunk 1]
```diff
@@ -52,96 +55,651 @@
...
-           <widget class="QLabel" name="timeLabel">
-            <property name="sizePolicy">
-             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
-              <horstretch>0</horstretch>
-              <verstretch>0</verstretch>
-             </sizepolicy>
-            </property>
-            <property name="font">
-             <font>
-              <family>Arial black</family>
```
Comment (lasconic) [inline mscore/playpanel.ui]: I suggest you don't change the font.

**Comment by ZackTheCardshark** [comment]
The first problem I see is that the tempo BPM and percentage are actually separated from the tempo slider—is it possible to do away with the grouping box and move those closer to the slider?

The second is the cluster of buttons at the bottom—I think the original suggestion of slightly separating the metronome and count-in buttons off to the right was a good one, even if you don't add a forward button.

Below is @etranger's original design concept on the right.

original design

**Comment by salilkhandelwal** [comment]
Well I can do something about the first problem . The second problem where you state to separate the metronome and count-in button , I tried that but it was looking awkward as it was disturbing the symmetry .

**Comment by ZackTheCardshark** [comment]
I think it still looks better than all five buttons in a group:

**Comment by ZackTheCardshark** [comment]
I think it still looks better than all five buttons in a group:

play

**Comment by ericfont** [comment]
The reason lasconic's image doesn't have the tempo number & percent near the slider has nothing to do with the fact that he is using a mac.  The real reason is because @salilkhandelwal needs to add *horizontal spacer* ui element between the posLabel & tempoLabel, and between the timeLabel & relTempoBox, like below:

need-horizontal-spacer

That way, when the window is expanded horizontally, then those elements will be pushed apart.

**Comment by ericfont** [comment]
(sorry if you had already figured that out...I couldn't tell.)

And like Zack said, should probably put those tempoLable and relTempoBox outside of the grouping box, to futher seperate the position ui elements from the tempo ui elements.

**Comment by salilkhandelwal** [comment]
Yes   @ZackTheCardshark  I  will be changing it today . @ericfont  We did not figure out that spacers were causing the problem .
 Another doubt that I have in my mind is that the maximum size of PlayPanelBase which is set to 16777215 X 16777215 , this is  making the layout awkward when we are stretching it out (even the old one as well) , I would like to suggest that we fix the maximum size of  PlayPanelBase to something less .

**Comment by salilkhandelwal** [comment]
screenshot from 2016-03-17 14 52 08 . This is how  play panel is looking now , any suggestions ?

File: mscore/playpanel.ui (hunk 1 above)
Comment (salilkhandelwal) [inline mscore/playpanel.ui]: font has been changed to Arial black

**Pull request closed without merging**